sdk/include/aditof/camera.h

```diff
@@ -529,6 +529,14 @@ class SDK_API Camera {
     loadDepthParamsFromJsonFile(const std::string &pathFile,
                                 const std::string &frameType) = 0;
 
+    /**
+     * @brief Reset the ini parameters from the chip and sets the ones stored in CCB
+     * @param mode - Camera mode to be reset
+     * @return Status
+     */
+    virtual aditof::Status
+    adsd3500ResetIniParamsForMode(const uint16_t &mode) = 0;
```

Comment:
Since mode is just an int, it works fine without using a reference

Reply:
Fixed.